### PR TITLE
Another attempt of pull request with partition by clause injected into buildColumnNames

### PR DESCRIPTION
**Pull request by eugene-frb**

### CODE REVIEW

**Comment by slnode** [comment]
Can one of the admins verify this patch? To accept patch and trigger a build add comment ".*ok\W+to\W+test.*"

File: lib/mssql.js [hunk 1]
```diff
@@ -342,15 +342,39 @@ function buildLimit(limit, offset) {
...
   if (filter.limit || filter.offset || filter.skip) {
     var orderBy = this.buildOrderBy(model, filter.order);
     var orderClause = '';
+    var partitionByClause = '';
+    if(filter.where) {
+      partitionByClause = this.buildPartitionBy(model, filter.where);
+    }
     if (orderBy) {
-      orderClause = 'OVER (' + orderBy + ') ';
+      orderClause = 'OVER (' + partitionByClause + ' ' + orderBy + ') ';
```
Comment (raymondfeng) [inline lib/mssql.js]: I'm not sure why you need to replace `ORDER BY` with `PARTITION BY`. The purpose was to support pagination for early version of SQL servers that don't support `OFFSET-FETCH` clause. See http://www.sqlshack.com/sql-server-2012-introduction-pagination/.

**Comment by eugene-frb** [comment]
Please replace this request with its correct version at https://github.com/strongloop/loopback-connector-mssql/pull/60.

I explained the need for PARTITION BY there.

**Comment by bajtos** [comment]
Closing in favour of #60.

**Pull request closed without merging**